specification/eventgrid/resource-manager/Microsoft.EventGrid/preview/2020-01-01-preview/EventGrid.json

```diff
@@ -2767,6 +2751,20 @@
           "description": "The base URL that represents the endpoint of the destination of an event subscription.",
           "type": "string",
           "readOnly": true
+        },
+        "maxEventsPerBatch": {
+          "format": "int32",
```

Comment:
are there min/max values that could be defined in swagger?

Reply:
Thanks for the review!  Currently the limits are enforced by our RP.  Once we close on the exact values, we will add the min/max values in swagger for our stable version.